test/shared/utils.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+var assert = require('assert');
+var R = require('ramda');
+
+var S = require('../..');
```

Comment:
Hmmm, I was sure I fixed these already. I even ran the extra line script just before I did my last push.